griptape/utils/stream.py

```diff
@@ -41,16 +41,6 @@ class Stream:
 
     structure: Structure = field()
 
-    @structure.validator  # pyright: ignore[reportAttributeAccessIssue]
-    def validate_structure(self, _: Attribute, structure: Structure) -> None:
-        from griptape.tasks import PromptTask
-
-        streaming_tasks = [
-            task for task in structure.tasks if isinstance(task, PromptTask) and task.prompt_driver.stream
-        ]
-        if not streaming_tasks:
-            raise ValueError("Structure does not have any streaming tasks, enable with stream=True")
-
```

Comment:
Not needed anymore?

Reply:
This is kind of lazy on my end, but the validation was no longer so straightforward. Now there are non-Prompt Task/Drivers that can stream events. Do you think having this validation is important?

Reply:
i dont think it matters too much. maybe we can update the docs to say "only tasks/whatever" that support a `stream` parameter will do anything, else it just executes normally?